### PR TITLE
Search and cart tests 

### PR DESCRIPTION
**Pull request by DashaLevchenko**

### CODE REVIEW

File: src/test/java/cart/test/CartTests.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                .header("Cookie", "PrestaShop-a30a9934ef476d11b6cc3c983616e364=gGUlJmz608BKmaascatCZmW2XTCt%2BH5QMbHSLUVItTlJR8MeJay6Kyud9egx8tfOW2FmjXodifkyRVtyklrSUotMhNs8o5BfUdr6opUEEhx%2B3bA6%2FgeEkyr7WPWQvmsD01ma%2ByT4%2F4DCbT5%2FTQfgnjtgzml7%2FLOm8S9cZEgkYdB88v6SPbNxsq1uLtdOxMFj000134")
+                .accept("application/json").contentType("application/x-www-form-urlencoded")
+                .body("controller=cart&" +
+                        "add=1&" +
+                        "ajax=true&" +
+                        "qty=1&" +
+                        "id_product="+id+"&" +
+                        "token=e817bb0705dd58da8db074c69f729fd8")
+                .when()
+                .post(URL.DEFAULT_URL);
```
Comment (4etverochka) [inline src/test/java/cart/test/CartTests.java:32]: move request to another class

File: pom.xml [hunk 2]
```diff
@@ -71,37 +71,21 @@
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
             <version>5.6.2</version>
-            <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
             <version>5.6.2</version>
-            <scope>test</scope>
```
Comment (yuliiakholina) [inline pom.xml:80]: what's wrong with scope?:)

File: src/main/java/cart/Cart.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import constants.URL;
+import io.restassured.response.Response;
+
+public class Cart {
+
+    public Response postAddCartResults(String id) {
+        return  RestAPIConnection.connection("application/x-www-form-urlencoded")
+                .header("Cookie", "PrestaShop-a30a9934ef476d11b6cc3c983616e364=gGUlJmz608BKmaascatCZmW2XTCt%2BH5QMbHSLUVItTlJR8MeJay6Kyud9egx8tfOW2FmjXodifkyRVtyklrSUotMhNs8o5BfUdr6opUEEhx%2B3bA6%2FgeEkyr7WPWQvmsD01ma%2ByT4%2F4DCbT5%2FTQfgnjtgzml7%2FLOm8S9cZEgkYdB88v6SPbNxsq1uLtdOxMFj000134")
+                .accept("application/json")
+                .body("controller=cart&" +
```
Comment (yuliiakholina) [inline src/main/java/cart/Cart.java:13]: Could we move response to separate file like json or at least to String value and just set qty, id and token?

File: src/main/java/connection/RestAPIConnection.java [hunk 4]
```diff
@@ -8,8 +8,12 @@ private RestAPIConnection(){
 
     }
 
-    public static RequestSpecification connection(){
+    public static RequestSpecification connectionHTML(){
         return RestAssured.given()
                 .header("Content-Type", "text/html");
     }
+    public static RequestSpecification connection(String contentType){
```
Comment (yuliiakholina) [inline src/main/java/connection/RestAPIConnection.java:15]: good decision here:)

**Pull request closed without merging**